packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -127,10 +132,15 @@ export class RestAPIClient {
 	 *
 	 * @param {EndpointVersions} version Version of the API to use.
 	 * @param {string} endpoint REST API path.
+	 * @param {EndpointNamespace} [namespace] REST API namespace.
 	 * @returns {URL} Full URL to the endpoint.
 	 */
-	getRequestURL( version: EndpointVersions, endpoint: string ): URL {
-		const path = `/rest/v${ version }/${ endpoint }`.replace( /([^:]\/)\/+/g, '$1' );
+	getRequestURL(
+		version: EndpointVersions,
+		endpoint: string,
+		namespace: EndpointNamespace = 'rest'
```

Comment:
By default, most requests that we need to work with are in the `rest` namespace except to delete invites from the site.